### ISSUE TITLE
[docs] Update RELEASE.md for generating container tests for the slsa-verifier CLI

### ISSUE THREAD

**Issue by asraa** [slsa-framework/slsa-github-generator#1110]
**Describe the bug**
Add a description and link to it in the RELEASE.md for how to generate verifier CLI tests when publishing new builders that release the container workflow.

See https://github.com/slsa-framework/slsa-github-generator/pull/1108

See issue on creating workflows: https://github.com/slsa-framework/example-package/issues/115

**Comment by ianlewis** [comment]
@asraa It looks like we haven't completed the e2e/regression tests for the container workflow in the verifier? We should probably finish them soon since the container workflow is technically GA in slsa-github-generator v1.4.0. Any specific issues that were still outstanding?
https://github.com/slsa-framework/slsa-verifier/blob/b06fbf5b0432d858b5576736ca6d793ee50b3199/cli/slsa-verifier/main_regression_test.go#L733-L792

**Comment by asraa** [comment]
Will take a look tomorrow -- I'm not sure why they're still commented out. 

Since we don't have `v1.4.0` in any of the tests (!!!), I can go ahead and add those and see if everything works correct.

**Comment by ianlewis** [comment]
> Since we don't have `v1.4.0` in any of the tests (!!!)

Yes. I think I meant to look into adding tests for the container workflow after I released v1.4.0 but it slipped my mind...
